docs/source/quickstart.rst

```diff
@@ -1,90 +1,193 @@
-Quick start
+Quick Start
```

Comment:
This is so, so good.  If I was looking at DDA for the first time I could be kicking tires in about 10 minutes.  Thank you so much for putting such thought into this.